.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,29 @@
+name: Lighthouse Report
+
+on: [pull_request]
+
+jobs:
+  lhci:
+    name: Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: npm install, build
+        run: |
+          npm install
+          npm run build
+      - name: run Lighthouse CI
+        run: |
+          npm install -g @lhci/cli@0.10.x
+          lhci autorun
+      - name: upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: Lighthouse reports
+          path: ${{ github.workspace }}/lighthouseReport
+          if-no-files-found: ignore
+          retention-days: 5
```

Comment:
remove `if-no-files-found` let it defaults to `warn`? retention 1 day

Reply:
ok